orbit/utils/simulation.py

```diff
@@ -53,14 +54,14 @@ def make_seasonality(series_len, seasonality, order=3, duration=1, scale=.05, ty
         in non-fourier type of seasonality ONLY.
     scale: float
         scale parameter of seasonality generation
-    type: str ['discrete', 'fourier']
+    method: str ['discrete', 'fourier']
     seed: int
     Returns
     -------
       np.array-llike with length equals `series_len`
```

Comment:
Please follow numpy docstring style guide for proper API docs parsing with sphinx https://numpydoc.readthedocs.io/en/latest/format.html